neurolang/probabilistic/tests/test_marg_query_resolution.py

```diff
@@ -28,8 +32,14 @@
 
 
 @pytest.fixture(
-    params=((weighted_model_counting, dichotomy_theorem_based_solver)),
-    ids=["SDD-WMC", "dichotomy-Safe query"],
+    params=(
+        (
+            weighted_model_counting,
+            dichotomy_theorem_based_solver,
+            dalvi_suciu_lift,
+        )
+    ),
+    ids=["SDD-WMC", "dichotomy-Safe query", "dalvi_suciu_lift"],
```

Comment:
what is "dichotomy-safe"?

Reply:
I don't know, I think you initially wrote that, no? I'm renaming this to `small-dichotomy`

Reply:
done